app/views/pulls/_merge.html.erb

```diff
@@ -22,6 +22,24 @@
     <% else %>
       <strong><%= l(("label_merge_" + pull.merge_status + "_title").to_sym) %></strong>
       <em class="info"><%= l(("label_merge_" + pull.merge_status + "_help").to_sym) %></em>
+
+      <% if pull.merge_status == 'conflicts_detected' && @conflicting_files&.length %>
+        <hr>
+
+        <p>
+          <strong>
+            <%= l(:conflicting_files) %>
+          </strong>
+        </p>
+
+        <ul>
```

Comment:
Can you adjust the margin on this? Maybe add `.pulls__merge :last-child { margin-bottom: 0; }`? What do you think?

![image](https://user-images.githubusercontent.com/1497697/47218293-43650e80-d379-11e8-8682-0366a9e05da7.png)